spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/integration/inbound/PubSubInboundChannelAdapter.java

```diff
@@ -179,6 +181,10 @@ private void logWarning(
     } else {
       LOGGER.warn(re.getMessage());
     }
+    // Log the stacktrace for troubleshoot.
+    StringWriter stringWriter = new StringWriter();
+    re.printStackTrace(new PrintWriter(stringWriter));
+    LOGGER.warn(stringWriter.toString());
```

Comment:
This would duplicate the log entry since we're already sending a log above. Can we adjust the log messages above, instead? For example, `LOGGER.warn("Message", exception)`

Reply:
Ah, thanks, I wasn't aware there's a `LOGGER.warn(String, Exception)` method.